.github/workflows/android_workflow.yaml

```diff
@@ -73,6 +73,12 @@ jobs:
       - name: Assemble app APK (debug)
         run: ./gradlew assembleDebug
 
+      - name: Upload APK
+        uses: actions/upload-artifact@v3
+        with:
+          name: apk
+          path: /home/runner/work/TaskMuse/TaskMuse/app/build/outputs/apk/debug/app-debug.apk
+
```

Comment:
Are these changes already included in @psavarmattas PR for login fields?

Reply:
Yes they are they are authored by sandesh in my branch.

Reply:
So we can rebase this branch with main and I'll approve

Reply:
We can merge it directly also it won't create a conflict